app/models/server/models/_BaseDb.js

```diff
@@ -282,6 +282,24 @@ export class BaseDb extends EventEmitter {
 		return result;
 	}
 
+	insertEphemeral(record, ...args) {
+		this.setUpdatedAt(record);
+
+		const result = this.originals.insert(record, ...args);
+
+		record._id = result;
+		const query = { _id: result };
+		const records = this.model.find(query).fetch();
```

Comment:
Why not a `findOne`? 👀 (since you're filtering by ID on the query, you can even use findById)